example_client/stateful/asr_demo/README.md

```diff
@@ -133,3 +133,75 @@ When the command finishes successfully you should see the `txt` file in the same
 cat /opt/workspace/sample.wav.txt
 /opt/workspace/sample.wav today we have a very nice weather
 ```
+
+### 7. Automatic speech recognition
+You can also run the live-demo.py client on windows machine to record wav files in real time with your microphone and send them to a ssh enabled server with mentioned kaldi and ovms containers setup.
```

Comment:
Perhaps we should remove mentioning "real time"? As the recognition does not happen in the real time and it might be confusing.